src/conda/env_eulerian_storm_track.yml

```diff
@@ -0,0 +1,16 @@
+name: _MDTF_eulerian_storm_track
+channels:
+  - anaconda
+  - conda-forge
+  - defaults
+dependencies:
+  - python=3.7
+  - matplotlib=3.2.2
+  - cartopy
+  - basemap
+  - numpy 
+  - netCDF4
+  - nco 
+  - xarray
+  - mock
```

Comment:
Based on your notes, mock should no longer be required for the POD. If it is, we can add it to the base yaml